scripts/release/init-all.sh

```diff
@@ -39,6 +39,7 @@ init_software()
     ${ROOT}/management_api/init.sh
     ${ROOT}/webrtc_agent/install_deps.sh
     ${ROOT}/video_agent/install_deps.sh
+    ${ROOT}/streaming_agent/install_ffmpeg.sh --enable-srt
```

Comment:
Is the '--enable-srt' option mandatory in init script?

Reply:
Yes, it is mandatory in streaming agent for SRT